.github/workflows/test-indexer-rpm.yml

```diff
@@ -0,0 +1,33 @@
+name: Test Wazuh Indexer upon upgrade - RPM
+on:
+  pull_request:
+    paths:
+      - 'stack/indexer/rpm/wazuh-indexer.spec'
+  workflow_dispatch:
+
+jobs:
+  Test-rpm-indexer-upgrade:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get the release version of Wazuh (all components)
+        run: echo "RELEASE_VERSION=$(jq -r 'map(select(.prerelease == false and .draft == false)) | .[] | .tag_name' <<< $(curl --silent https://api.github.com/repos/wazuh/wazuh/releases) | sed "s|v||g" | sort -rV | head -n 1)" >> $GITHUB_ENV
+      - name: Get the major and minor of the release version
+        run: echo "MAJOR_MINOR=$(echo $RELEASE_VERSION | cut -d '.' -f1-2 | sed "s|\.||")" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v3
+      - name: Build the Wazuh indexer package and set env variable
```

Comment:
Indexer should be capitalized and the variable maybe should be changed in the comment to `set environment variable`

Reply:
Done.